app/javascript/components/batch_update/components/TagInput.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { colors } from './Layout'
+
+export const PENDING = {
+  ADD: 'ADD',
+  REMOVE: 'REMOVE',
+}
```

Comment:
There is perhaps a more elegant way to do this?

My main concern here was to not rely on [stringly-typed](http://wiki.c2.com/?StringlyTyped) props.